public/electron.js

```diff
@@ -123,7 +123,7 @@ function createWindow() {
     ipcMain,
     lndSettingsDir,
     lndPort: LND_PORT,
-    macaroonsEnabled: MACAROONS_ENABLED,
+    network: NETWORK,
```

Comment:
`network: isDev ? 'simnet' : NETWORK`
Else the app will use `'testnet'` in dev mode as well.